cpp/src/copying/contiguous_split.cu

```diff
@@ -160,7 +176,7 @@ __device__ void copy_buffer(uint8_t* __restrict__ dst,
                             std::size_t element_size,
                             std::size_t src_element_index,
                             uint32_t stride,
-                            int value_shift,
+                            int64_t value_shift,
```

Comment:
*question*: This is trivially a positive change, but to my understanding (cc @nvdbaranec) we still need to fix up the copy/adjustment that is now happening around lines 195->210. Is this correct?

Reply:
Yes, I think so. I mentioned this in the PR description.

Reply:
Correct.